src/Storage/Storage.Management/Az.Storage.psd1

```diff
@@ -192,7 +192,9 @@ CmdletsToExport = 'Get-AzStorageAccount', 'Get-AzStorageAccountKey',
                'Set-AzStorageBlobTag', 'Get-AzStorageBlobTag', 
                'Get-AzStorageBlobByTag', 
                'Invoke-AzStorageAccountHierarchicalNamespaceUpgrade', 
-               'Stop-AzStorageAccountHierarchicalNamespaceUpgrade'
+               'Stop-AzStorageAccountHierarchicalNamespaceUpgrade', 
+               'Set-AzStorageBlobImmutabilityPolicy', 'Remove-AzStorageBlobImmutabilityPolicy',
+               'Set-AzStorageBlobLegalHold', 'Invoke-AzRmStorageContainerImmutableStorageWithVersioningMigration'
```

Comment:
Is there any way we can shorten the name by using abbreviations as alias?

Reply:
We have discussed with PM for that, and finally get not short it.